spec/factories/cohorts.rb

```diff
@@ -16,7 +16,11 @@
     end
 
     trait :with_funding_cap do
-      funding_cap { true }
+      after(:build) do |cohort|
+        next unless cohort.start_year >= 2024
```

Comment:
I think best avoid the 2024 value and just go with setting the funding cap to true false to determine if that cohort is funding cap ready as not keen on having hard values. Let me know your thoughts